### PR TITLE
Welinton

### PR DESCRIPTION
**Pull request by deleonw**

### CODE REVIEW

**Comment by deleonw** [comment]
Filter Button Displays Filter Menu

File: mealsfromdirt/package-lock.json
Comment (EllieMP) [inline mealsfromdirt/package-lock.json:1]: This file should bot be deleted

File: mealsfromdirt/package.json
Comment (EllieMP) [inline mealsfromdirt/package.json:1]: This file should not be deleted

**Review by EllieMP — CHANGES_REQUESTED** [review]
The package.json and package-lock.json files should not be deleted.  I’m not sure if this was a mistake.

**Pull request closed without merging**